app.vue

```diff
@@ -1,6 +1,20 @@
 <template>
-  <div>
+  <div
+    class="prose max-w-none
+    prose-h1:type-headline-1
+    prose-h2:type-headline-2 prose-h2:border-t prose-h2:pt-16 prose-h2:scroll-mt-16 prose-h2:lg:scroll-mt-8
+    prose-h3:type-headline-3 prose-h3:pt-8 prose-h3:scroll-mt-24 prose-h3:lg:scroll-mt-16
+    "
+  >
     <Header />
     <NuxtPage />
   </div>
 </template>
+
+<style scoped>
+.prose :where(h2 > a, h3 > a) {
```

Comment:
This isn't ideal **but** our the markdown content design has some links underlined (paragraphs) and some not (headings).